src/main/python/pythonparser/pythonparser_3.py

```diff
@@ -1,16 +1,21 @@
+#!/usr/bin/env python3
+
 # Copyright (c) Raychev, V., Bielik, P., and Vechev, M. (see https://eth-sri.github.io/py150)
 # Copyright (c) Victor Quach (see https://github.com/Varal7/pythonparser)
-# !/usr/bin/env python3
 
-import sys
+
 import json as json
 import ast
 import asttokens
 from xml.sax.saxutils import quoteattr
 import argparse
+from typing import List, Dict, Union
+
+ListOfDicts = List[Dict]
+ListOfAstNodes = List[ast.AST]
```

Comment:
I think you can use it directly because it is not a difficult type. It is a good way to make a new type if you have a big type annotation like `List[Dict[MyClass, List[int]]]`. Add types in `Dict`, please. For example

`def traverse_list(l: List[ast.AST], node_type: str = 'list', node: ast.AST = None) -> int:`

`def parse_file(filename: str) -> List[Dict[key_type, value_type]]`

Reply:
Thanks!
Is type `Dict[str, Union[str, List[int]]]` deserves its own type alias? 
Something like `JsonNodeType = Dict[str, Union[str, List[int]]]` 
What do you think?

Reply:
I think yes, it is a great idea to use the special variable for it